src/7-to-8/major-changes/platforms.rst

```diff
@@ -0,0 +1,98 @@
+Platforms
+=========
+
+.. note::
+
+   The terms :term:`platform` and :term:`job platform` are equivelent.
```

Comment:
s/equivelent/equivalent

Reply:
Started reviewing when I noticed this file had been changed in #255, see comment above :point_up:

Reply:
probably not.
Oops.